gnoi/os/client.go

```diff
@@ -55,3 +55,16 @@ func (c *Client) Activate(ctx context.Context, version string) error {
 	}
 	return nil
 }
+
+// Verify invokes the Verify RPC for the OS service and it returns the running version and activation fail message or an error.
+func (c *Client) Verify(ctx context.Context) (string, string, error) {
```

Comment:
One of the few cases it is worth naming the return variables.

```
func (c *Client) Verify(ctx context.Context) (version, failMsg string, err error) {
        var out *pb.VerifyResponse
	if out, err = c.client.Verify(ctx, &pb.VerifyRequest{}); err != nil {
		return
	}
	version = out.GetVersion()
        failMsg = out.GetActivationFailMessage()
	return
}
```

Reply:
Oh that's a great idea